### ISSUE TITLE
Multistart on different ports of the same IP

### ISSUE THREAD

**Issue by VacheLorian** [Jigsaw-Code/outline-server#1080]
Hey! Can you please tell me how can I modify install_server.sh to run multiple Outline containers on different ports of the same IP?

I have tried `sudo CONTAINER_NAME=shadowbox-inst3 SHADOWBOX_DIR=/opt/outline/inst3 -E install_server.sh`. But the problem is towerwatch. It is recreated and already created servers cannot be managed in the Outline Manager.

**Comment by bemasc** [comment]
I don't think you can do this using the "dockerized" version of Outline via install_server.sh, for the reason you've identified.  However, you can probably do this manually by starting multiple instances of Outline without Docker (see [here](https://github.com/Jigsaw-Code/outline-server/tree/master/src/shadowbox#running-shadowbox-as-a-nodejs-app)).

A single Outline instance can listen on multiple ports.  If that's sufficient for your use case, it's likely easier to manage.
